helm/cluster-cloud-director/values.yaml

```diff
@@ -95,6 +95,10 @@ nodePools: {}
   #   class: default  # Node classes name (defined above)
   #   replicas: 2
 
+ntp:  # ntp pools and servers chrony config
+  servers: []
```

Comment:
Can we add some default values here? is it possible?

Reply:
If we don't set anything, the default `chrony` pools are applied (currently what happens). 
The rest of the config will be the same as I just matched it.

```
pool ntp.ubuntu.com        iburst maxsources 4
pool 0.ubuntu.pool.ntp.org iburst maxsources 1
pool 1.ubuntu.pool.ntp.org iburst maxsources 1
pool 2.ubuntu.pool.ntp.org iburst maxsources 2
```